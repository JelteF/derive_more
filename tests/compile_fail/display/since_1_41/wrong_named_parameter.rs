#[derive(derive_more::Display)]
#[display(fmt = "Stuff({bars})")]
pub struct Foo {
    bar: String,
}

fn main() {}
