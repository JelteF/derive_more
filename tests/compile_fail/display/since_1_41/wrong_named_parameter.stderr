error[E0425]: cannot find value `bars` in this scope
 --> tests/compile_fail/display/since_1_41/wrong_named_parameter.rs:2:17
  |
2 | #[display(fmt = "Stuff({bars})")]
  |                 ^^^^^^^^^^^^^^^ help: a local variable with a similar name exists: `bar`
